front/app/modules/commercial/content_builder/admin/components/ContentBuilderFrame/index.tsx

```diff
@@ -35,6 +35,8 @@ const ContentBuilderFrame = ({ projectId }: { projectId: string }) => {
         canvas
         style={{
```

Comment:
Let's take these styles out of here and add them to the RenderNode (only is the node is root). That way' they won't get serialized by the editor and won't affect the actual preview.